sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/checks/XmlSchemaCheck.java

```diff
@@ -228,12 +232,16 @@ public void setSchemas(String schemas) {
   }
 
   private void validate() {
-    if ("autodetect".equalsIgnoreCase(schemas)) {
-      autodetectSchemaAndValidate();
-    } else {
-      String[] schemaList = StringUtils.split(schemas, " \t\n");
-
-      validate(schemaList);
+    try {
+      if ("autodetect".equalsIgnoreCase(schemas)) {
+        autodetectSchemaAndValidate();
+      } else {
+        String[] schemaList = StringUtils.split(schemas, " \t\n");
+        validate(schemaList);
+      }
+    } catch (SchemaNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)